src/TableContent.js

```diff
@@ -28,9 +58,11 @@ const TableContent = ({tableHeader, searchedColumnIndex, searchQuery, orderedCol
                 <div>
                   {
                     checkedRows.length > 0 ?
-                      <a href={URI(`experiments/download/zip`, host).search({accession: checkedRows}).toString()}>
-                      Download {checkedRows.length} {checkedRows.length === 1 ? `entry` : `entries`}
-                      </a> :
+
+                      <a className={`downloadButton`} onClick={() => alertInvalidFiles(`json/experiments/download/zip/check`, host, checkedRows)}>
```

Comment:
As with the previous function `alertInvalidFiles` already implies the endpoiint you’re passing as first argument. Or put it another way: you don’t call `alertInvalidFiles` with any other argument other than `json/experiments/download/zip/check` as first argument. That’s a constant value that should be part of the body of the function.